src/adapters/cloudflare/mod.rs

```diff
@@ -16,6 +17,15 @@ use responses::CloudflareResponse;
 
 use crate::artifacts::CloudflareManifest;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct WorkerScript {
+    pub id: String,
+    pub created_on: String,
+    pub modified_on: String,
+    pub usage_model: Option<String>,
+    pub etag: String,
```

Comment:
I can't remember what we do elsewhere... is it better to derive `Getters` here? I'm usually hesitant when there are a bunch of public fields on a struct. Additionally, if we hand-write the getters, we can return an `&str` instead of a `String`, which is more ergonomic and eliminates frequent cloning.

Not blocking.